_summaries/chen2020a_2.md

```diff
@@ -1,19 +1,50 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: chen2020a
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: 7
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
```

Comment:
The paper has many ablations, which could be good material for this "interesting variants" section. Since you already mention the conclusions of these ablations (e.g. this network benefits relatively more from larger batch sizes), you could just include the relevant graphs for batch size/epochs, network size, augmentation combinations, etc. instead of creating another section.